app/views/layouts/application.html.erb

```diff
@@ -47,5 +47,15 @@
         <%= yield %>
       </div>
     </div>
+    <footer class="footer">
+      Your IP address is:
+      <% if Rails.env.production? %>
+        <%= request.remote_ip %>
+      <% else %>
+        <%= Net::HTTP.get(URI.parse(URL_TO_CHECK_IP_ADDRESS)).squish %>
```

Comment:
nice thought.
But show only the remote_ip in all environments.
What will happen if there is no internet connection